docs/source/intervention_models/maternal_supplementation/index.rst

```diff
@@ -456,12 +456,121 @@ In the baseline scenario, the exposure distribution of birthweight (mean birthwe
+
+Child wasting
+~~~~~~~~~~~~~
+
+This modeling strategy is intended to work in tandem with the :ref:`dynamic transition model of child wasting <2020_risk_exposure_wasting_state_exposure>`. The effect of birthweight improvements due to maternal supplementation on child wasting exposure will be applied to the wasting state that the simulant is initialized into. We will conservatively assume that birthweight improvements due to maternal supplementation does not have an impact on x-factor exposure status and/or wasting exposure transition rates.
+
+For each gram increase in a simulant's birthweight due to a maternal supplementation intervention (:math:`S`), the category 1 (severe wasting/SAM) and category 2 (moderate wasting/MAM) exposures used to determine the probability of initialization into those states should be reduced proportionately such that the total reduction in moderate and severe wasting exposure prevalence is equal to 0.0115 / 200 = 0.0000575. The exposure prevalence of category 3 (mild wasting) should be increased by 0.0115 / 200 = 0.0000575. The figure below demonstrates how to implement this change visually. 
+
+For the :ref:`acute malnutrition simulation <2019_concept_model_vivarium_ciff_sam>`, the impact of maternal supplementation interventions on CGF exposures can be implemented for simulants born into the simulation only given the six month burn-in period.
+
+.. note::
+
+  For baseline calibration of IFA coverage and wasting initialization state:
+
+    The :math:`S` shift applied to the wasting initialization probabilities according to baseline IFA coverage should be the following:
+
+      uncovered = -(IFA_bw_shift * baseline_IFA_coverage)
+
+      uncovered = -(IFA_bw_shift * baseline_IFA_coverage) + IFA_bw_shift
+
+  Then, the :math:`S` shift in the intervention scenario should be equal to the sum of all maternal supplementation intervention impacts on birthweight.
+
+.. image:: wasting_exposure_dist.svg
+
+Assumptions and limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The application of the size from [McGovern-et-al-2019-maternal-supplementation]_ makes the following assumptions:
+
+- The effect size is entirely causal and not subject to confounding
+
+- The effect between BW and wasting measured among children under five is applied to prevalent wasting status at six months of age only and does not affect future wasting exposure trajectories (aside from any associated vicious cycle effects). This is a conservative underestimation of the impact of birthweight on child wasting burden.
```

Comment:
Is there any way to have a simple *over*estimate, since I expect this effect to be disappointingly small?

Reply:
I see two pathways to do this:

1. have the BW-->wasting impact affect the wasting transition rates such that the steady state is set to the have X percent less wasting exposure for all age groups 6-59 months (I think this would have approximately a 5% reduction in wasting exposure prevalence from baseline compared to 20-30% reduction for the direct interventions).
2. update x-factor exposure based on improved wasting state at initialization (this will have less of an impact than #1). 

Both of these strategies are limited in that they do not consider effect modification by age because it was not quantified by McGovern despite that claim that the effect is stronger in the younger ages and less pronounced in older children.

Reply:
@aflaxman, just want to make sure you see this!

Reply:
Let's circle back to it when we have seen the impact with the model you have proposed here.

Reply:
Ok, sounds good. Thanks